app/views/medical_records/_form.html.erb

```diff
@@ -31,7 +31,16 @@
     <%= form.text_field :current_state %>
   </div>
 
+  <div>
+    <%= form.label :notes, style: "display: block" %>
+    <%= form.text_area :notes, rows: 5 %>
+  </div>
+
   <div>
     <%= form.submit %>
   </div>
+  <% if @medical_record.image.attached? %>
+    <%= image_tag("Odontogram.jpg", alt: "odontograma", class:"d-block w-100") %>
+  <% end %>
```

Comment:
You don't need to have an image attached to the medical record to show an image that is part of your repository.